lib/urlwatch/jobs.py

```diff
@@ -230,3 +231,29 @@ def retrieve(self, job_state):
                 return response.content.decode('ascii', 'ignore')
 
         return response.text
+
+class FileJob(Job):
+    """Read the content of a local file"""
+
+    __kind__ = 'file'
+    __required__ = ('path',)
+
+    def get_location(self):
+        return str(self.path)
+
+    def retrieve(self, job_state):
+
+        path_string = str(self.path)
+
+        if not path_string:
+            raise ValueError('Path cannot be null or empty')
+
+        if path_string.startswith('file://'):
```

Comment:
Unfortunately it seems that requests library doesn't automatically support `file://` or `ftp://` protocols. The only available option would be to define a custom adapter to support each different protocol. More information here:

- http://docs.python-requests.org/en/master/user/advanced/#transport-adapters
- http://stackoverflow.com/questions/10123929/python-requests-fetch-a-file-from-a-local-url

What do you think would be the best solution?